autoapi/mappers/python.py

```diff
@@ -146,6 +152,14 @@ def args(self):
     def args(self, value):
         self._args = value
 
+    @property
+    def docstring(self):
+        return self._docstring
+
+    @docstring.setter
```

Comment:
Where does this come from?

Reply:
`self.docstring` is set in the constructor already. I've had to turn this into a property so that `PythonClass` can customise how this is accessed and use the `__init__` docstring if configured.